src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1420,6 +1417,7 @@ def __init__(
             self.path = []
         else:
             self.path = path
+        self._static_info = None  # command's static info
```

Comment:
`__full_static_info` stores static info for all datamodels, starting from root at class-level. `_static_info` stores the static-info at command level which is passed with `create_instance`.